jdaviz/app.py

```diff
@@ -349,50 +350,68 @@ def _color_to_level(color):
                                       history=msg_level >= history_level,
                                       popup=msg_level >= popup_level)
 
-    def _link_new_data(self):
+    def _link_new_data(self, reference_data=None, data_to_be_linked=None):
```

Comment:
where does this get called?  I can't seem to find any calls to `_link_new_data` in this diff or anywhere in `jdaviz` or `glue-jupyter` 🤔

Reply:
Maybe here?

https://github.com/spacetelescope/jdaviz/blob/ff6e66886c85610320c8e846cd3cc2527d46a2a5/jdaviz/app.py#L1258

Reply:
🤦 thanks! Clearly my "search entire project" is buggy, since it didn't come back with that for some reason.  For future reference, this is triggered by `DataCollectionAddMessage`.

Reply:
Usually I use the good old recursive `grep` on local checkout when in doubt. 😉